src/capital-project/capital-project.controller.ts

```diff
@@ -33,6 +36,15 @@ import { ZodTransformPipe } from "src/pipes/zod-transform-pipe";
 export class CapitalProjectController {
   constructor(private readonly capitalProjectService: CapitalProjectService) {}
 
+  @UsePipes(new ZodTransformPipe(findCapitalProjectsQueryParamsSchema))
+  @Get("/")
+  async findCapitalProjects(
```

Comment:
Okay, I know this discussion is buried and following it is tedious ...but...naming conventions on controllers/controllers/and repositories...

[Discussion on naming conventions](https://github.com/NYCPlanning/ae-zoning-api/discussions/115#discussion-5982504)

In this case, the guidelines would say to use `findMany` instead of `findCapitalProjects`. This applies to the controller, service, and repository.

@TylerMatteo , I'm not sure how much you think we should enforce this.